Scripts/SQLServer/BackfillScripts/Backfill_22_10.sql

```diff
@@ -0,0 +1,17 @@
+-- 1.backfill JobOrganizationUnitId
+with CTE as (
+select qie.[JobOrgUnitFullyQualifiedName] as qOrgName, qie.[JobOrganizationUnitId] as qOrgId, j.[OrganizationUnitId] as jOrgId, j.[OrgUnitFullyQualifiedName] as jOrgName
+from [dbo].[QueueItemEvents] qie 
+inner join [dbo].[QueueItems] qi on qie.[QueueItemId] = qi.[Id]
+inner join [dbo].[Jobs] j on qi.[ExecutorJobId] = j.[Id]
+where j.[OrganizationUnitId] != qie.[JobOrganizationUnitId] or (qie.[JobOrganizationUnitId] is null and j.[OrganizationUnitId] is not null))
+update CTE
+set qOrgName = jOrgName, qOrgId = jOrgId;
+
+--2.truncate read table
+truncate table [read].[QueueItemEvents];
```

Comment:
how do we make sure that there's no MigrateJobs/QueueItems already running when we run this script?

Reply:
truncate wait the other statement finish before executing. I started insert before truncate. Truncate doesn't start until insert finish.